ionic2/app/models/mob.ts

```diff
@@ -2,15 +2,16 @@ import { POI } from './poi';
 
 export class Mob extends POI {
   clusterId: number;
-  tweets: PokeTweet[];
+  tweets: MobTweet[];
   timestamp: number;
 
   type = 'mob';
 }
 
-export type PokeTweet = {
+export type MobTweet = {
```

Comment:
If we extract this class to a separate file and call it `Tweet` we can reuse it in other parts of the application. For instance, the sentiment analysis also returns an array of tweets and these are not related to tweets.